### DISCUSSION TITLE
Customize the steps component

### DISCUSSION THREAD

**Discussion by FeiPengDev** [vueform/builder#81] [Questions] [answered]
Excuse me, is there a way to customize the Steps component?
For example, I want to arrange Steps vertically, I tried it myself but couldn't find a solution.
![image](https://github.com/user-attachments/assets/8553421d-1986-450b-b4c6-b9bce7a042ab)

**Comment by adamberecz** [comment, marked as answer]
Unfortunately there's no option to do that currently, however it's in our roadmap: https://github.com/vueform/vueform/discussions/79
